Makefile

```diff
@@ -1,18 +1,13 @@
-.PHONY: all build unit-tests integration-tests
+.PHONY: build unit-tests integration-tests
 
 override GOOS:=$(shell uname)
 override GO111MODULE=on
 
-all: build
-
 unit-tests:
-	@go test ./...
+	@CGO_ENABLED=0 go build && go test ./...
 
 integration-tests:
 	@go test -tags=integration ./...
 
-build: unit-tests
```

Comment:
Can we update the readme to reflect these changes?
We can remove [this](https://github.com/ExpediaGroup/mittens/blob/a81c8765e20f100aa9638f3b6b09ae2619c9b663/README.md#build-binary-executable) and rename [this](https://github.com/ExpediaGroup/mittens/blob/a81c8765e20f100aa9638f3b6b09ae2619c9b663/README.md#run-unit-tests) to build & run.